krun/util.py

```diff
@@ -180,3 +181,15 @@ def print_session_info(config):
             print("  %s" % k)
     else:
         print("  No keys skipped")
+
+
+def make_heat():
+    """A loop which spins, attempting to make heat.
+
+    Used when the CPU is too cool to run a benchmark."""
+
+    until = time.clock() + 1  # in roughly one second
+    while True:
+        # Waste cycles
+        if time.clock() > until:
```

Comment:
re: I/O doesn't matter since we sync, but I prefer your loop anyway if it works.